packages/react/src/components/contact-form/ContactFormDetails.tsx

```diff
@@ -146,17 +147,22 @@ export const ContactFromDetails = ({ data, editing }: { data: Contact, editing:
         </ItemForm>
 
         <ItemForm>
-          <FormTextbox
+          <NumberBox
```

Comment:
I see that you use number-box here but we also use FormTextbox could we replace it with text-box from devextreme? Also, it is an incorrect name. Should be FormTextBox